docs/development_suite/overview-dev-suite.md

```diff
@@ -4,28 +4,47 @@ title: Console Overview
+* Building and launching digital products **quickly**.
+* Viewing market changes as **opportunities**.
+* Being **flexible** to rapidly respond to demand fluctuations.
+* **Integrating** with various tools, partners, and ecosystems.
 
-Modern companies build their digital products, data and logics through a Digital Platform.  
-The tool to reach this goal is the **Internal Developer Portal** which will help you to industrialize and govern cloud-native development and operations. It allows to:
+Modern companies achieve this through an Internal Developer Platform (IDP). The tool to accomplish this is **Mia-Platform Console**, an Internal Developer Platform builder which aids in industrializing, managing and simplifying cloud-native development and operations.
 
-* Design and Expose APIs;
-* Develop and Orchestrate microservices;
-* Manage Real-time Data Streams and Event-driven architectures;
-* Govern and monitor runtime on K8s.
+![Mia-Platform Console Overview](img/Mia-Platform-infographic-overview.png)
 
-Mia-Platform Console allows you to govern all projects in one place, industrialize & automate DevOps, avoid organizational bottlenecks, self-serve developers, solve cloud complexity and enhance accountability & reliability.
+## Why Mia-Platform Console is important
 
-**Hierarchical Organization of Mia-Platform Console:**
+Mia-Platform Console manages the entire software lifecycle seamlessly and unify your IT department under one integrated platform. Simplify cloud-native development while maintaining complete governance and control over your processes.
+
+* **Clear Governance**: Establish streamlined, standardized pathways for software delivery and integrate governance rules directly into your development workflows, ensuring consistency, compliance, and efficiency throughout the entire development process.
+* **Self-service Approach**: Build and manage an Internal Developer Platform (IDP) that empowers development teams by providing them with the necessary tools and resources in a self-service mode, enabling them to develop efficiently and autonomously without dependency on other departments.
+* **Lower cognitive load**: Abstracting underlying complexities simplifies tasks for developers, minimizing cognitive load and reducing the need for frequent context switching. This enables them to concentrate more on creative problem-solving and efficient coding, ultimately enhancing productivity and innovation within development projects.
+
+## Ease the digital platform development
+
+Mia-Platform Console serves the IT team involved in software development because it brings several advantages to the development of an Internal Developer Platform.
+
+- **Governance & controls**: Manage runtime resources across hybrid and multi-cloud architectures, connect clusters, establish self-service environments, enforce RBAC policies, and establish project templates that standardize developer tools, by creating 'paved roads'.
```

Comment:
I think we should also highlight here the ability to choose and connect your own providers such as Git, pipelines, etc.

Reply:
Here is my proposal:
Manage runtime resources across hybrid and multi-cloud architectures, connect clusters, establish self-service environments and enforce RBAC policies, by creating 'paved roads'. You can also establish project templates that standardize developer tools and the ability to choose and connect your own provider such as Git.